socialite/server/config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+	uri: "mongodb+srv://SRDewan:abcd1234@database.vvxaz.mongodb.net/Data?retryWrites=true&w=majority"
```

Comment:
does this file need to be uploaded? or should it be in .gitignore?

Reply:
i am not sure about this although

Reply:
Needed, this contains the mongodb link which is used for connecting to atlas
Later on u can use this file to store other global variables.